src/components/static-map.tsx

```diff
@@ -48,73 +64,144 @@ export function StaticMap() {
   }
 
   return (
-    <div className="h-screen bg-slate-700">
-      <h1>Map Demo</h1>
-      <div id="map-container" className="grid grid-cols-2 gap-5">
-        <div id="map" className="">
+    <div>
+      <h1 className="mt-8 pl-5 text-4xl font-semibold">Mapa Wydarzenia</h1>
+      <hr className="mb-16 mt-5" />
+      <div id="map-container">
+        <div className="mx-auto mb-10 hidden w-[90%] sm:grid sm:grid-cols-2 sm:gap-2 xl:grid-cols-4">
+          <MapFloorsButton
+            onClick={() => {
+              switchMapView("Outside");
+            }}
+            active={outside}
+            text="Scena plenerowa, gastronomia, atrakcje"
+            icon={<SunIcon className="size-10" />}
+          />
+          <MapFloorsButton
+            onClick={() => {
+              switchMapView("Floor 1");
+            }}
+            active={floorOne}
+            text="Trybuna A, strefa dla gości"
+            icon={<h1 className="text-3xl font-bold">1</h1>}
```

Comment:
Lepiej nie uzywac naglowkow h1 do jakichs ikon, tekkstow etc., naglowek powinien byc jeden na poczatku strony, uzywanie niezgodnie z przeznaczeniem miesza np w czytnikach.

Reply:
To tu na przykład dać p?